src/test/java/com/redislabs/redistimeseries/RedisTimeSeriesTest.java

```diff
@@ -246,6 +246,21 @@ public void testAdd() {
       Assert.assertEquals(600L, client.add("seriesAdd", 600L, 1.1, 10000, null));
     }
 
+    Assert.assertEquals(
+        400L,
```

Comment:
Can we test for a duplicate timestamp? I believe 400L is not a duplicate one correct?

Reply:
The entire `seriesAdd4` key is new here. 
Also, I did not find any duplicate timestamp example to follow/get an idea.

Reply:
@sazzad16 example of on_duplicate in ts.add

# using cli

```
redis-cli ts.add serie1 1 10.0
redis-cli ts.add serie1 1 5.0 ON_DUPLICATE MAX
redis-cli ts.range serie1 - +
```
Expected result:
```
$ redis-cli ts.add serie1 1 10.0
(integer) 1
$ redis-cli ts.add serie1 1 5.0 ON_DUPLICATE MAX
(integer) 1
$ redis-cli ts.range serie1 - +
1) 1) (integer) 1
   2) 10
```

# examples of go client
https://godoc.org/github.com/RedisTimeSeries/redistimeseries-go#example-Client-AddWithOptions-DuplicateDatapointsMaxDuplicatePolicy
https://godoc.org/github.com/RedisTimeSeries/redistimeseries-go#example-Client-AddWithOptions-DuplicateDatapointsLastDuplicatePolicy

Reply:
@filipecosta90 I've added one before seeing your reply. Could you please check that version?

Reply:
:+1: